### ISSUE TITLE
'Options' object has no attribute 'module_name'

### ISSUE THREAD

**Issue by tufedtm** [django-inplaceedit/django-inplaceedit#76]
python 2.7.6 (venv)
1 install django 1.8
2 install django-inplaceedit
3 add model
```python
class News(models.Model):
    title = models.CharField(max_length=50)
    text = models.TextField(max_length=2000)
```
4 add view
```python
def indexy(request):
    news = News.objects.get(pk=1)
    context = {
        'news': news
    }
    return render(request, 'base.html', context)
```
5 add template
```html
{% load inplace_edit %}

<!DOCTYPE html>
<html>
<head lang="en">
  <meta charset="UTF-8">
  <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"/>
  <title></title>
{% inplace_toolbar %}
</head>

<body>

{% inplace_edit "news.title" %}
<br/><br/>
{{ news.text }}

</body>
</html>
```
on load http://127.0.0.1:8000 show this
![2015-06-03 17 42 00](https://cloud.githubusercontent.com/assets/4018676/7962769/e9993472-0a17-11e5-94fd-7a529b0077ad.png)
![2015-06-03 17 42 14](https://cloud.githubusercontent.com/assets/4018676/7962770/e999db98-0a17-11e5-8fb1-392963861384.png)

**Comment by mhoffa** [comment]
### As an option writer is not correct you can fix at the file fields.py ###

**Replace it with my line**

```python
line 57: self.config ['module_name'] = self.model._meta.model_name
```
and
```python
line 106: return modelform_factory (self.model, exclude = [])
```

**Comment by tufedtm** [comment]
replace it
```python
self.config['obj_id'] = unicode(self.obj.pk)
```
with this
```python
self.config ['module_name'] = self.model._meta.model_name
```
and it
```python
return config
```
with this
```python
return modelform_factory (self.model, exclude = [])
```

now show this
type object 'AdaptorTextField' has no attribute 'model'
![2015-06-06 12 22 24](https://cloud.githubusercontent.com/assets/4018676/8019111/b73afa7a-0c46-11e5-9a4a-03cc2bf7db88.png)

**Comment by mhoffa** [comment]
Two days ago, I needed his library I corrected only two lines, and I run the project. I see you try to set Pycharm of his library catch error, just as I did.

**Comment by mhoffa** [comment]
that I confused you, see who is doing something for this pool of requests

https://github.com/Yaco-Sistemas/django-inplaceedit/pull/72/files

**Comment by tufedtm** [comment]
@mhoffa 
thanks!

**Comment by goinnn** [comment]
Fixed here: 9fcdff3fe0b724869ec09476a9aea566de48bab9
